documentation/asciidoc/stories/assembly_creating_services.adoc

```diff
@@ -18,6 +18,9 @@ include::{topics}/ref_persistent_cache_store.adoc[leveloffset=+2]
 include::{topics}/proc_allocating_cpu_memory.adoc[leveloffset=+1]
 include::{topics}/proc_setting_jvm_options.adoc[leveloffset=+1]
 
+include::{topics}/proc_disabling-fips-mode-cr.adoc[leveloffset=+1]
```

Comment:
Hi @dvagnero .

Would you know how I can conditionalize this file for the downstream docs? I cannot find the attribute.